project-words

```diff
@@ -0,0 +1,85 @@
+activestorage
```

Comment:
I generated the list by running `cspell` then used GPT to turn the output into a list, but there may be a better way.